modules/images/dominant-color/load.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Module Name: Dominant Color
+ * Description: Adds support to store dominant-color for an image and create a placeholder background with that color.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Add the dominant color metadata to the attachment.
+ *
+ * @param array $metadata The attachment metadata.
+ * @param int   $attachment_id The attachment ID.
+ *
+ * @return array $metadata
+ */
+function dominant_color_metadata( $metadata, $attachment_id ) {
+	if ( ! wp_attachment_is_image( $attachment_id ) ) {
+
+		return $metadata;
+	}
+	$dominant_color = dominant_color_get( $attachment_id );
+
+	if ( ! empty( $dominant_color ) ) {
+		$metadata['dominant_color'] = $dominant_color;
+	}
+
+	return $metadata;
```

Comment:
Blank lines in this function are a bit confusing and doesn't make too much sense. Could you please avoid adding blank lines where they aren't needed and group related code together to make it easier to read and understand it? Same applies to all functions and methods in this PR.
```suggestion
	if ( ! wp_attachment_is_image( $attachment_id ) ) {
		return $metadata;
	}

	$dominant_color = dominant_color_get( $attachment_id );
	if ( ! empty( $dominant_color ) ) {
		$metadata['dominant_color'] = $dominant_color;
	}

	return $metadata;
```